pkg/controller/controller.go

```diff
@@ -184,6 +187,15 @@ func (c *controller) Run(workers int, stopCh <-chan struct{}) {
 		}
 	}
 
+	c.clusterID = string(uuid.NewUUID())
```

Comment:
If the service catalog process gets restarted, does it need/want to get a new UUID? seems like cluster id will be recreated each time we start-up the service catalog.

Reply:
it will read from the configmap if it exists and use the configmap value.